.github/workflows/veracode.yaml

```diff
@@ -0,0 +1,128 @@
+###############################################################
```

Comment:
Please still remove the veracode workflow

Reply:
removed